EIPS/eip-6122.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: 6122
+title: Forkid checks based on timestamps
+author: Marius van der Wijden (@MariusVanDerWijden)
+description: Modifies the forkid checks to work with timestamps and block numbers
+discussions-to: https://ethereum-magicians.org/t/eip-6122-forkid-checks-based-on-timestamps/12130
+status: Draft
+type: Standards Track
+category: Networking
+created: 2022-12-13
+requires: 2124
+---
+
+## Abstract
+
+[EIP-2124](./eip-2124.md) proposed a way of identifying nodes on the p2p network based on their chain configuration via the forkid parameter. 
+It allows nodes to cut incompatible nodes off quickly which makes the P2P network more reliable.
+After the merge, forks are scheduled by blocktime instead of blocknumber. This EIP updates the forkid calculation with blocktime.
```

Comment:
```suggestion
After the merge, forks are scheduled by block time instead of block number. This EIP updates the forkid calculation with block time.
```